src/module-vsbridge-indexer-catalog/ArrayConverter/Product/CustomOptionConverter.php

```diff
@@ -92,8 +92,8 @@ private function filterData(array $option): array
     {
         $option = $this->dataFilter->execute($option, $this->fieldsToDelete);
         $option['sort_order'] = (int) $option['sort_order'];
-        $option['option_id'] = (int) $option['sort_order'];
-        $option['option_type_id'] = (int) $option['sort_order'];
+        $option['option_id'] = (int) $option['option_id'];
+        $option['option_type_id'] = (int) $option['option_type_id'];
```

Comment:
option_type_id seems not to be set in every case...I fixed by changing to:
`$option['option_type_id'] = (isset($option['option_type_id'])) ? (int) $option['option_type_id'] : 0;`

Reply:
Ok, thank you. Added suggestion